src/main/java/seedu/address/logic/commands/group/DeleteGroupCommand.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.logic.commands.group;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.exceptions.GroupNotFoundException;
+
+/**
+ * Command to delete a new group with the specified name and persons inside.
+ */
+public class DeleteGroupCommand extends Command {
+    public static final String COMMAND_WORD = "deleteGroup";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes the group with the specified name.\n"
+            + "Parameters: GROUP_NAME\n"
```

Comment:
I am not so sure about this one but should a prefix be used for the `GROUP_NAME`?

Reply:
Yep, will edit this to keep consistent with the CreateGroupCommand